guides/adr/0012-formalize-accredited-provider-relationships.md

```diff
@@ -0,0 +1,380 @@
+# 12. Formalize Accredited Provider relationships
+
+Date: 23/10/24
+
+## Status
+
+Proposal
+
+## Context
+
+
+### Evolving provider relationships
+The structural and conceptual relationships among and between providers and users has changed in many ways since this codebase was started.[[1]]
+
+Providers have always had a relationship between one another in the form of course accreditation. More recently, this relationship has been extended to delegating this accreditation.
```

Comment:
For clarity, DfE accredits providers, and these accredited providers ratify courses published by their training (a.k.a. lead) partners.

Accredited providers cannot delegate their accreditation to training partners.